Formula/universal-ctags.rb

```diff
@@ -0,0 +1,57 @@
+class UniversalCtags < Formula
+  desc "Maintained ctags implementation"
+  homepage "https://github.com/universal-ctags/ctags"
+  url "https://github.com/universal-ctags/ctags/archive/refs/tags/p5.9.20210523.0.tar.gz"
+  sha256 "00e5493454c7c013b996a24019b8f8cbd75968e1993fb56f7e2737b7309cc684"
+  license "GPL-2.0-only"
+  head "https://github.com/universal-ctags/ctags.git"
```

Comment:
```suggestion
  head "https://github.com/universal-ctags/ctags.git"

```
Let's try to follow the format of other formulae here.